GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -2759,6 +2777,7 @@ protected override CommandStatus ExecuteCommand(int cmd)
                 case Commands.CompareToCurrentBranch: CompareWithCurrentBranchToolStripMenuItem_Click(null, null); break;
                 case Commands.CompareToBranch: CompareToBranchToolStripMenuItem_Click(null, null); break;
                 case Commands.CompareSelectedCommits: compareSelectedCommitsMenuItem_Click(null, null); break;
+                case Commands.OpenCommitsWithDifftool: diffSelectedCommitsMenuItem_Click(null, null); break;
```

Comment:
This should be turned into `diffSelectedCommitsMenuItem.PerformClick()` then.

Reply:
If so, also existing should be changed

Reply:
This specific call was changed in the second commit